modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -988,7 +959,7 @@ private CompletableFuture<Void> startPartitionAndStartClient(
                         );
 
                         SnapshotStorageFactory snapshotStorageFactory = createSnapshotStorageFactory(replicaGrpId,
-                                partitionUpdateHandlers, internalTable);
+                                partitionUpdateHandlers, internalTbl);
```

Comment:
Seems that we no longer use abbreviations. No sure though.

Reply:
`internalTbl` variable was declared before my changes, I can rename it but it will introduce more changed lines

Reply:
Ok, Let it be `internalTbl`.